tools/Status/Status.py

```diff
@@ -190,7 +190,9 @@ def _state_order(state):
 
 
 def _format(field: str, value: Any, state_styles: dict) -> str:
-    if field == "State":
+    if pd.isnull(value) or value == "None assigned":
```

Comment:
Which field can print as "None assigned"? Could you add a comment?

Reply:
The `NodeList` slurm filed returns "None assigned" if the job is pending. I'll add a comment